imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -65,17 +65,13 @@ def initialize(self) -> None:
         self.mf6 = Mf6Wrapper(
             lib_path=self.dfm_metamod_config.kernels.modflow6.dll,
             lib_dependency=self.dfm_metamod_config.kernels.modflow6.dll_dep_dir,
-            working_directory=copy.deepcopy(
-                self.dfm_metamod_config.kernels.modflow6.work_dir.absolute()
-            ),
+            working_directory=self.dfm_metamod_config.kernels.modflow6.work_dir,
```

Comment:
now you can remove the "import copy " above

Reply:
Done